### DISCUSSION TITLE
shall I change 'trigger frequence' on 'Chainlink upkeeper'?

### DISCUSSION THREAD

**Discussion by frank0532** [Cyfrin/foundry-full-course-cu#90] [Q&A] [answered]
I have create a 'upkeeper' on ChainLink and trigger a function of smartcontract once per hour, and until now it works well.
Now I want to change its 'trigger frequence' from 'once per hour' to 'once per day', but I can't find how to do this change.

**Comment by alymurtazamemon** [comment, marked as answer]
@frank0532 Cancel this one and register a new one with a different cron job.

here you will find your active scheduler: https://automation.chain.link/
